### PR TITLE
Update style.scss

### PR DESCRIPTION
**Pull request by yukulele**
hard to read content while denim.png is not loaded

### CODE REVIEW

**Comment by arthurvr** [comment]
This shouldn't be a separated rule, it should be done using the `@include` for Compass '`background()`, like this:

```scss
@include background(
    radial-gradient(20...),
    ...
    image-url('denim.png'),
    $dark
)
```

@yukulele Could you update your PR to do so? Thanks for the PRs!

**Comment by yukulele** [comment]
done

File: src/sass/style.scss [hunk 1]
```diff
@@ -329,8 +329,8 @@ table {
 
 html {
   background-image: url('img/denim.png');
-  color: $lightest;
   background-color:  $dark;
```
Comment (arthurvr) [inline src/sass/style.scss]: of course you can remove this line now :) Thanks for being that quick updating the PR!

Comment (arthurvr) [inline src/sass/style.scss]: The background mixin now takes care of that.

**Comment by arthurvr** [comment]
Merged. Thanks @yukulele! All contributions are welcome!

**Pull request closed without merging**